<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndustryCheck</class>
 <widget class="QWidget" name="IndustryCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1249</width>
    <height>943</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>IndustryCheck</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>941</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox_0">
    <property name="geometry">
     <rect>
      <x>932</x>
      <y>10</y>
      <width>260</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="connectDeviceBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QLabel" name="deviceNameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>设备名：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="deviceComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>912</width>
      <height>608</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>912</width>
       <height>608</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>912</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>911</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>932</x>
      <y>120</y>
      <width>260</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>图像采集</string>
    </property>
    <widget class="QLabel" name="captureModelabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>采集模式：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="captureModecomboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="triggerModelabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>触发模式：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="triggerModecomboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="captureStateBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>开始采集</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settempRoiBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>设置模板ROI</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>932</x>
      <y>320</y>
      <width>260</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>图像处理</string>
    </property>
    <widget class="QLabel" name="binarizationThreshold">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>二值化阈值：</string>
     </property>
    </widget>
    <widget class="QSlider" name="binarizationSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>220</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="binarizationthresholdNum">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="morphkernal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>形态学核心：</string>
     </property>
    </widget>
    <widget class="QLabel" name="morphkernalNum">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="morphkernalSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>220</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="canny">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Canny边缘：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cannyEditNum1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>45</width>
       <height>25</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="cannyNum1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>阈值1：</string>
     </property>
    </widget>
    <widget class="QLabel" name="cannyNum2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>阈值2：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cannyEditNum2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>200</y>
       <width>45</width>
       <height>25</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="contoursArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>过滤轮廓面积：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="contoursAreaEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>280</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="saveParameterBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>设置参数</string>
     </property>
    </widget>
    <widget class="QLabel" name="contoursLength">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>过滤轮廓长度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="contoursLengthEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="tempMatch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>模板匹配：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tempMatchRoi">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>匹配模板ROI</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>932</x>
      <y>720</y>
      <width>260</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="geometricDetection">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>几何检测：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="detectionBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>开始检测</string>
     </property>
    </widget>
    <widget class="QLabel" name="geometricCalculation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>几何计算：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="calculationBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>开始计算</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1202</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
